defs.bzl

```diff
@@ -0,0 +1,23 @@
+def gmaven_artifact(fqn):
+  parts = fqn.split(":")
+  packaging = "aar"
+
+  if len(parts) == 3:
+    group_id, artifact_id, version = parts
+  elif len(parts) == 4:
+    group_id, artifact_id, version, packaging = parts
```

Comment:
I believe the orders for length 4 and 5 are
groupId:artifactId:packaging:version
groupId:artifactId:packaging:classifier:version
https://maven.apache.org/pom.html#Maven_Coordinates

Reply:
I remembered following the order in that page, but it was switched out about a year ago: https://github.com/bazelbuild/bazel/commit/7d5b31f9cb33dd86542cd2cd0fea63954010fc00

Reply:
Done. Switched to follow `groupId:artifactId:packaging:version` and `groupId:artifactId:packaging:classifier:version`